adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -816,18 +816,7 @@ private IWindowComponent wrapActivity(final Activity activity) {
             throw new IllegalArgumentException("activity");
         }
 
-        return new IWindowComponent() {
-            private Activity mRefActivity = activity;
```

Comment:
Not exactly. The `new IWindowComponent() { ... }` class holds a reference to the outer `AuthenticationContext` class as long as it is still in scope. Because of this, the `activity` parameter passed into `wrapActivity` will also still be in scope.